### DISCUSSION TITLE
Comment Feature

### DISCUSSION THREAD

**Discussion by YoMamasNerd** [Tkd-Alex/Twitch-Channel-Points-Miner-v2#470] [Ideas]
Hey,
could you impliment a feature that randomly comments something to make you appear more active in chat?

**Comment by Tkd-Alex** [comment]
Nope, sorry

**Reply by YoMamasNerd** [reply]
Just out of curiosity - would it be possible to hook into your code in run.py to post comments in the chat?

**Reply by Tkd-Alex** [reply]
mhh no
